app/mailers/work_mailer.rb

```diff
@@ -4,6 +4,6 @@ class WorkMailer < ActionMailer::Base
   def untouched_works_notification(work_ids)
     @works = Work.where(id: work_ids)
 
-    mail(to: 'anannict@gmail.com', subject: '【Annict::Marie】未更新の作品を更新して下さい')
+    mail(to: 'anannict@gmail.com', subject: '【Annict DB】未更新の作品を更新して下さい')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.